src/main.go

```diff
@@ -138,6 +139,13 @@ func main() {
 		showcommands()
 	case "eksexpiry":
 		eksexpiry.Expiry()
+	case "aerolab-ansible":
+		os.Args = []string{}
+		os.Args = append(os.Args, "aerolab")
+		os.Args = append(os.Args, "inventory")
+		os.Args = append(os.Args, "ansible")
```

Comment:
Replace the 4 lines with (first param in case someone called it as ./aerolab instead of aerolab, or in case of some people ./al and other variations):
```
os.Args = []string{os.Args[0],"inventory","ansible"}
```